Update where to check for existing chapters (#136)
rladies/rladiesguide: content/coordination/onboarding/index.en.md

## Goal

The goal of this activity is to start a conversation with prospective

> organizers, answer any questions and send information on how to get a

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Author: Thanks for letting me know.

Reviewer 2: ```suggestion
organisers, answer any questions and send information on how to get a
```
I'll reverse the spelling to UK so we can merge these changes an update the Guide

    Team, the messages to this account are forwarded to the email
    address you provided during your own onboarding process.

> -   On the rladies.org website, we ask any prospective organizers to get

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

    -   Name(s) of organiser(s) and their email address (to be added to
        the organisers slack).

-   Make a search on GitHub
    ([https://github.com/rladies/rladies.github.io/tree/main/data/chapters](https://github.com/rladies/rladies.github.io/tree/main/data/chapters))
    to make sure the city does not have a chapter yet. You can use CTRL+F to search using the city's name.

> -   If there is a chapter already, check the name of the organizers

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reviewer 2: ```suggestion
-   If there is a chapter already, check the name of the organisers
```

        -   Country

>         -   Organizers

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reviewer 2: ```suggestion
        -   Organisers
```

        -   Status = Prospective

> -   Invite organizers to the organizers slack.

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reviewer 2: ```suggestion
-   Invite organisers to the organisers slack.
```

> -   Send the prospective organizer an email with all the most important

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reviewer 2: ```suggestion
-   Send the prospective organiser an email with all the most important
```

    information and links (see [Appendix A](#appendix-a))

-   **Other types of requests:**

>     -   People may ask to be added to the organizer slack, please check

Reviewer 2: ```suggestion
    -   People may ask to be added to the organiser slack, please check
```

>         if their name is in the meetup list of co-organizers. If not,

Reviewer 2: ```suggestion
        if their name is in the meetup list of co-organisers. If not,
```

        ask why.

>     -   Change list of organizers :\

Reviewer 2: ```suggestion
    -   Change list of organisers :\
```

>         Check and amend the list of organizers here:\

Reviewer 2: ```suggestion
        Check and amend the list of organisers here:\
```

>         Email the former organizer for confirmation (see [template B](#appendix-b)).

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

>         Email the former organizer for confirmation (see [template B](#appendix-b)).

Reviewer 2: ```suggestion
        Email the former organiser for confirmation (see [template B](#appendix-b)).
```

you will find the
R-Ladies Global leadership team. Feel free to reach out to them through
the channel #ask_the_leadership if you have questions that are not

> addressed in the Starter Kit, or the Guidelines, or that other organizers

Reviewer: ```suggestion
addressed in the Guidelines or that other organisers
```

Starter-Kit no longer really contains relevant information, so we shouldnt mention it.

> cannot answer. More info about the R-Ladies Global team is here:

Reviewer: ```suggestion
cannot answer. More info about the R-Ladies Global team is can be found on the R-Ladies website:
```

## Appendix B

> > confirm list of organizers - email template

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reviewer 2: ```suggestion
> confirm list of organisers - email template
```

```markdown
Hi \<\<first name\>\>,

Thanks for sending through the updated list of organizers. I sent the

> slack invites to the new organizers.

Reviewer 2: ```suggestion
slack invites to the new organisers.
```

We keep information about chapters and organizers on this list on
GitHub:
https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv.

-   Could you please send name and surname of the new organizers so that
    I can update the chapter list?

-   Could you also please confirm whether I should remove the following

>     from the list of organizers?

Reviewer 2: ```suggestion
    from the list of organisers?
```

    -   \<\<NAME1\>\>

    -   \<\<NAME2\>\>

We assume former organizers provide all the necessary information to new

> organisers before passing the baton :)

Reviewer: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.
